src/Sulu/Bundle/ContentBundle/Resources/public/js/components/content/settings/main.js

```diff
@@ -377,6 +380,26 @@ define([
                 this.sandbox.dom.attr('#shadow_on_checkbox', 'checked', true);
                 this.sandbox.emit('husky.toolbar.header.item.disable', 'state', false);
             }
+
+            var def = $.Deferred();
+
+            // set author form data
+            this.sandbox.form.create('#settings-author-container').initialized.then(function() {
```

Comment:
Is it a good idea to mix the husky-validation forms and the manual handling? We also have that in other places, and it confuses me quite a lot 😕